core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -6701,7 +6702,7 @@ class ReplicaManagerTest {
       def haltProcedure(exitStatus: Int, message: Option[String]): Nothing = {
         fail("Test failure, broker should not have halted")
       }
-      Exit.setHaltProcedure(haltProcedure)
+      Exit.setHaltProcedure((status, message) => haltProcedure(status, Option(message)))
```

Comment:
nit: this can be inlined, now that it requires a lambda wrapper. The arguments can be `(_, _)` because the body doesn't depend on their values.

Reply:
Cool, it looks more concise, learn a trick 😺 
Thank you